vendor/github.com/v3io/v3io-tsdb/pkg/config/config.go

```diff
@@ -278,6 +278,11 @@ func GetOrLoadFromStruct(cfg *V3ioConfig) (*V3ioConfig, error) {
 	return instance, nil
 }
 
+// Eagerly reloads TSDB configuration
+func UpdateConfig(path string) {
+	instance, failure = loadConfig(path)
```

Comment:
Does anything ever check this failure?

Reply:
It might be a good idea to protect this with a lock, as this assignment is not atomic.